test_backward_compatibility.py

```
@@ -0,0 +1,217 @@
+This skill should still work exactly as before.
+''')
+            
+            properties = read_properties(skill_dir)
+            
+            # Basic properties should work
+            assert properties.name == "old-skill"
+            assert properties.description == "An existing skill without requirements"
+            assert properties.license == "MIT"
+            assert properties.compatibility == "all"
+            assert properties.when_to_use == "When you need to process files"
+            assert properties.disable_model_invocation == False
```

Ruff (0.15.13)

[error] Avoid equality comparisons to `False`; use `not properties.disable_model_invocation:` for false checks

Replace with `not properties.disable_model_invocation`

(E712)